src/components/header/Header.jsx

```diff
@@ -10,9 +10,21 @@ import {
 import "./Header.css";
 import { NavLink } from "react-router-dom";
 import { useAuthContext } from "../../context/AuthContext";
+import { useVideoContext } from "../../context/VideoContext";
```

Comment:
Suggestion: You can import both useAuthContext and useVideoContext from a single file